.travis.yml

```diff
@@ -24,4 +24,21 @@ deploy:
     script: bash scripts/deploy_docker.sh $TRAVIS_TAG
     on:
       tags: true
-      branch: main
\ No newline at end of file
+      branch: main
+
+jobs:
+  - name: Coverage
+    rust: nightly
+    script:
+      - curl --location https://github.com/mozilla/grcov/releases/latest/download/grcov-linux-x86_64.tar.bz2 | tar jxf -
+      # These flags are recommended by https://github.com/mozilla/grcov#grcov-with-travis .
+      # vk: I had to remove `-Zpanic_abort_tests -Cpanic=abort` because this would cause compilation
+      # to fail but I haven't investigated more into why.
+      # I added `-Awarnings` which allows all warnings. This was necessary because nightly can
+      # introduce some new warnings and when testing one such warning appeared in the ethcontract
+      # generated contract code which caused the whole file to be printed as part of the warning
+      # message which made the build fail (probably travis related).
+      - env CARGO_INCREMENTAL=0 RUSTFLAGS="-Zprofile -Ccodegen-units=1 -Copt-level=0 -Clink-dead-code -Coverflow-checks=off -Awarnings" cargo +nightly test
```

Comment:
I would prefer if we multi-line this: each env var becomes a `export XYZ=foo` and the execution of the tests would also be on a separate line (see  https://github.com/mozilla/grcov#grcov-with-travis)